From 703dc62e647900c62ffa0b0296bb90e90f48e933 Mon Sep 17 00:00:00 2001
Message-ID: <703dc62e647900c62ffa0b0296bb90e90f48e933.1699318203.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1699318202.git.sharlatanus@gmail.com>
References: <cover.1699318202.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 7 Nov 2023 00:30:22 +0000
Subject: [PATCH 16/16] gnu: cl-graph: Update to 0.10.2-2.c617de3.

* gnu/packages/lisp-xyz.scm (cl-graph): Update to 0.10.2-2.c617de3.
[source]: Update URL as maintainer and owner of repository was changed.

Change-Id: Ice484f098d52b93baebc9596f0f902b1b30ae797
---
 gnu/packages/lisp-xyz.scm | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index d4bf0e8bdf..3f1dcc8952 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -2704,8 +2704,8 @@ (define-public ecl-cl-dot
   (sbcl-package->ecl-package sbcl-cl-dot))
 
 (define-public sbcl-cl-graph
-  (let ((commit "3cb786797b24883d784b7350e7372e8b1e743508")
-        (revision "1"))
+  (let ((commit "c617de35390cb02db88bc5b5febffafdb8947ae8")
+        (revision "2"))
     (package
       (name "sbcl-cl-graph")
       (version (git-version "0.10.2" revision commit))
@@ -2713,11 +2713,11 @@ (define-public sbcl-cl-graph
        (origin
          (method git-fetch)
          (uri (git-reference
-               (url "https://github.com/gwkkwg/cl-graph")
+               (url "https://github.com/hraban/cl-graph")
                (commit commit)))
          (file-name (git-file-name "cl-graph" version))
          (sha256
-          (base32 "1748rj52f2jmd5jvsy9jwhn0zf73sjzjhwhnljvq6yi2kdqr30kl"))))
+          (base32 "0g1abkph9zb0m9zz6q2471ml9q9acdhwyapk8ra3bisqpwlxvpyf"))))
       (build-system asdf-build-system/sbcl)
       (arguments
        ;; TODO: (Sharlatan-20221118T215839+0000): Tests failed
-- 
2.41.0

